setup.py

```diff
@@ -74,6 +76,7 @@
     "SQLAlchemy~=1.3.0",
     "kafka-python~=1.4",
     "sklearn-crfsuite~=0.3.6",
+    "setuptools~=41.0.1",
```

Comment:
for tf 1.14 or t2t

Reply:
is there an issue on their GH or some reference? would be good to know, because this is a pretty heavy restriction I'd rather not impose. If they need a minimun 41, then we should rather go for `>=` on this one

Reply:
yeah, I think they need a minimum